paasta_tools/vitesscluster_tools.py

```diff
@@ -15,6 +17,594 @@
 from paasta_tools.utils import load_v2_deployments_json
 
 KUBERNETES_NAMESPACE = "paasta-vitessclusters"
+# Image variables
+IMAGE_TAG = "v16.0.3"
+VTCTLD_IMAGE = f"docker-paasta.yelpcorp.com:443/vitess_base:{IMAGE_TAG}"
+VT_GATE_IMAGE = f"docker-paasta.yelpcorp.com:443/vitess_base:{IMAGE_TAG}"
+VT_TABLET_IMAGE = f"docker-paasta.yelpcorp.com:443/vitess_base:{IMAGE_TAG}"
+VT_ADMIN_IMAGE = f"docker-dev.yelpcorp.com/vtadmin:{IMAGE_TAG}"
+
+
+# Global variables
+TOPO_IMPLEMENTATION = "zk2"
+TOPO_GLOBAL_ROOT = "/vitess-paasta/global"
+SOURCE_DB_HOST = "169.254.255.254"
+TABLET_TYPES = ["primary", "migration"]
+WEB_PORT = "15000"
+GRPC_PORT = "15999"
+
+# Mapping of mysql cluster to target role envoy port
+MYSQL_CLUSTER_PORT_MAP = {
```

Comment:
Will move this to srv-configs or some other internal repo

Reply:
++. This helps. Will move this as a config to puppet

Reply:
Fixed in https://github.com/Yelp/paasta/pull/3889/commits/a8e1e2a9c43a88c68bfd4a83862cde0c2a94f1da. Resolving